src/github.com/getlantern/flashlight/main/main.go

```diff
@@ -53,7 +54,12 @@ func init() {
 }
 
 func logPanic(msg string) {
-	cfg, err := config.Init(flashlight.PackageVersion, *configdir, *stickyConfig, flagsAsMap())
+	// Request the config via either chained servers or direct fronted servers.
+	// We have to do the full version here because we're actually trying to
+	// submit this panic to our logs.
+	cf := util.NewChainedAndFronted(client.Addr)
+	configFetcher := config.NewFetcher(settings.GetUserID, settings.GetToken, cf)
```

Comment:
The user ID is interesting though -- that's assigned by the pro server, is that correct @xiam? Does it also assign it to free users? Regardless it will get sent up whenever it's populated @oxtoacart, but we could trigger it too.